packages/blaze/materializer.js

```diff
@@ -95,7 +95,32 @@ const materializeDOMInner = function (htmljs, intoArray, parentView, workStack)
 
 const isPromiseLike = x => !!x && typeof x.then === 'function';
 
-function waitForAllAttributesAndContinue(attrs, fn) {
+function then(maybePromise, fn) {
+  if (isPromiseLike(maybePromise)) {
+    maybePromise.then(fn);
```

Comment:
I think we should `.catch` and pipe the error into the internal logging

Reply:
Do you mean `Blaze._reportException`? If so, I'd add it for all async-related code since right now those `Promise`s are not handled at all. I did it on purpose, since it results in a global error anyway, but sure, Blaze-specific logging would make more sense.

Reply:
Ping @jankapunkt.

Reply:
From what I understand by [this comment](https://github.com/meteor/blaze/pull/368#issuecomment-1489102811) then APM tools like Monti APM hook into Blaze's internal logging for their reporting etc.